_opcodes/op99-bgee.html

```diff
@@ -0,0 +1,33 @@
+---
+n: 99
+name: "Spell Effect: Duration Modifier"
+param1: "Duration Modifier"
+param2: "Type"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Alters spell durations of the targeted creature(s), to the percentage specified by the 'Duration Modifier' field, affecting the spell type specified by the 'Type' field. Timing mode 0 is the only timing mode reliably affected. Timings handled through PRO files are ignored.
+<br/>For instance:
+<ul>
+	<li>'Duration Modifier' = 0 &longrightarrow; Spells have no duration</li>
+	<li>'Duration Modifier' = 100 &longrightarrow; Spells have normal duration</li>
+	<li>'Duration Modifier' = 200 &longrightarrow; Spells have double duration</li>
+</ul>
+<br/>
+
+<br/>Known values for 'Type' are:
+<ul>
+	<li>0 &longrightarrow; Mage</li>
+	<li>1 &longrightarrow; Priest</li>
+	<li>2 &longrightarrow; Bard Song</li>
+</ul>
+
+{% capture note %}
+Durations handled by <a href="#op333">Static Charge</a> are unaffected.
```

Comment:
You can just add these two things to the shared version instead.

Reply:
Done.